gradle.properties

```diff
@@ -11,4 +11,5 @@
 sdk.dir=/home/matthes/opt/android/sdk
 org.gradle.jvmargs=-Xmx2048M
 android.useAndroidX=true
-android.enableJetifier=true
\ No newline at end of file
+android.enableJetifier=true
+MAPBOX_DOWNLOADS_TOKEN=sk.eyJ1IjoiamRiYXJhZCIsImEiOiJjbDIzMHU4NjgxbGh4M2NscGQ4ZDN2ZHJ2In0.KZxcRGwzBRraA39k9HaKcQ
```

Comment:
You should never ever publish any sensitive credentials on GitHub. I'd strongly suggest to change your token as soon as possible.

Reply:
It's Test Token